Tests/Resources/Fixtures/loader_config/valid7.yml

```diff
@@ -0,0 +1,12 @@
+stdClass:
+    definitions: 
+        - 
+            uri_schema: /forum/{category}/{post_title}/edit
+            defaults:
+                _type: edit
+        -
```

Comment:
I wonder what this results in? Definitions named "0" and "1" doesn't look great to me